Main.java

```diff
@@ -9,7 +11,31 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Asks the user to enter a number to start playing
+    System.out.println("Please enter a number to play FizzBuzz");
+
+    //Create interger for number
+    int num = input.nextInt();
+
+    //Write constant for a number divisible by 3
+    int DIVISIBLE_BY_3 = (num % 3);
+    
+    //Write constant for a number divisible by 5
+    int DIVISIBLE_BY_5 = (num % 5);
+
```

Comment:
Remember, UPPER_CASE_LETTERS should be for constants only. These could be constants but you would need to put the word final in front. Otherwise, this is great code. Very professional looking.